core/embed/prodtest/main.c

```diff
@@ -235,7 +249,9 @@ static void test_display(const char *colors) {
         c = 0xFFFF;
```

Comment:
also in this functions, proper color constants are needed to ensure function across different color depths.

Reply:
fixed, see 989fffebf2196106384103bd7800dfdcbb7a54b9